libs/portals/admin/form-system/src/components/MainContent/components/Urls/Urls.tsx

```
@@ -0,0 +1,137 @@
+                        if (e.target.checked) {
+                          handleCreateFormUrl(url.id)
+                        } else {
+                          handleDeleteFormUrl(url.id)
+                        }
+                      }
+                    }}
+                    name={`test-url-${url?.id}`}
+                    label={url?.url}
+                  />
+                </Box>
+              ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)